spring-vault-core/src/main/java/org/springframework/vault/core/lease/SecretLeaseContainer.java

```diff
@@ -592,11 +588,16 @@ protected Lease doRenewLease(RequestedSecret requestedSecret, Lease lease) {
 
 	@SuppressWarnings("unchecked")
 	private Lease renew(Lease lease) {
+		final Map<String, String> leaseRenewalData = new LinkedHashMap<>();
```

Comment:
I was looking for an ImmutableMap of some kind, but the Guava version was failing. I would be more than happy to change this